js/objects/tests/test-serialization-with-preload.js

```diff
@@ -30,7 +30,7 @@ describe("Serialization / Deserialization Tests", () => {
             let second = serializationEl.textContent;
             assert.equal(first, second);
         });
-        it("Deserializing from itself will produce an identical JSON serialization", function(){
+        it.skip("Deserializing from itself will produce an identical JSON serialization", function(){
```

Comment:
we should probably adjust this test, say check on everything other than the id property, since otherwise we are liable to create serialization issues

Reply:
Perhaps we should build two equality methods on Parts, as follows:
* Two parts are said to be "equal" if all of their properties -- id aside -- have the same values and they are of the same type;
* Two parts are said to be identical if the first item is true, but they also have the same id.
  
What we are expecting in this test is that they are equal but not identical